rsocket/framing/Frame.cpp

```diff
@@ -249,4 +249,25 @@ std::ostream& operator<<(std::ostream& os, const Frame_ERROR& frame) {
               << ", " << frame.payload_;
   }
 
+  StreamType getStreamType(FrameType frameType) {
+    if (frameType == FrameType::REQUEST_STREAM) {
+      return StreamType::STREAM;
+    } else if (frameType == FrameType::REQUEST_CHANNEL) {
+      return StreamType::CHANNEL;
+    } else if (frameType == FrameType::REQUEST_RESPONSE) {
+      return StreamType::REQUEST_RESPONSE;
+    } else if (frameType == FrameType::REQUEST_FNF) {
+      return StreamType::FNF;
+    } else {
+      CHECK(false) << "Unknown open stream frame : " << frameType;
```

Comment:
Nit: Use `LOG(FATAL)` instead.  It's marked as `[[noreturn]]` so there won't be any compilation errors.